linera-core/src/unit_tests/test_utils.rs

```diff
@@ -94,7 +90,6 @@ where
 pub struct LocalValidatorClient<S>
 where
     S: Storage,
```

Comment:
I believe these `S: Storage` constraints can also be removed now — they were only here to allow access to `S::StoreError`.

Reply:
Let me try now in a different PR

Reply:
No it doesn't work. It traces back to the implementation of SimpleObject for chain state views

Reply:
We can always replace `SimpleObject` with manual `#[Object]` getters. It's not as nice as the definition site but it doesn't infect the rest of the codebase

Reply:
There are a lot of subviews to manage. I guess we could fork and modify `#[derive(SimpleObject)]` eventually.